__tests__/app.test.js

```diff
@@ -200,3 +200,25 @@ describe("3. GET /api/reviews/:review_id", () => {
     });
   });
 });
+
+describe("5. GET /api/users", () => {
+  test("status: 200, responds with an array of objects that contain username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toBeInstanceOf(Array);
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toEqual(
+            expect.objectContaining({
```

Comment:
These two lines can be replaced with expect(user).toMatchObject({   // expect.any    })